### DISCUSSION TITLE
Welcome to WindowsAppSDK-Samples Discussions!

### DISCUSSION THREAD

**Discussion by btueffers** [microsoft/WindowsAppSDK-Samples#63] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Participate in [Windows App SDK discussions](https://github.com/microsoft/WindowsAppSDK/discussions) that aren't related to samples!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Spliffworks** [comment]
Hi I'm new here, still learning code, so not very helpful at mo. I hope I can learn a bit from anybody out there, and become more productive and be able to help more in the future. any pointers will help
